Userland/Applications/PixelPaint/Tools/Tool.h

```diff
@@ -77,6 +77,9 @@ class Tool {
 
     virtual StringView tool_name() const = 0;
 
+    void set_alt_key_override() { m_override_alt_key = true; } // We only set the override_alt_key flag to true since the override is false by default. If false is desired do not call method.
+    bool is_overriding_alt() { return m_override_alt_key; };
```

Comment:
Or alternatively make it a virtual function returning `false` by default and then override it only in the tools that have an alt mode